protocol/x/revshare/keeper/revshare_test.go

```diff
@@ -261,3 +266,320 @@ func TestValidateRevShareSafety(t *testing.T) {
 		})
 	}
 }
+
+func TestKeeper_GetAllRevShares_Valid(t *testing.T) {
+	tests := []struct {
+		name                              string
+		revenueSharePpmNetFees            uint32
+		revenueSharePpmTakerFees          uint32
+		expectedRevShares                 int
+		monthlyRollingTakerVolumeQuantums uint64
+		setup                             func(tApp *testapp.TestApp, ctx sdk.Context,
+			keeper *keeper.Keeper, affiliatesKeeper *affiliateskeeper.Keeper)
+	}{
+		{
+			name:                              "Valid revenue share",
```

Comment:
Nit: could we rename to more descriptive test case. E.g. `affiliate + unconditional + marketmapper`